smqtk_image_io/impls/image_reader/gdal_io.py

```diff
@@ -372,15 +374,17 @@ def get_config(self):
             'channel_order': self._channel_order,
         }
 
-    def valid_content_types(self):
+    def valid_content_types(self) -> Set[str]:
         """
         :return: A set valid MIME types that are "valid" within the implementing
             class' context.
         :rtype: set[str]
         """
         return get_gdal_driver_supported_mimetypes()
 
-    def _load_as_matrix(self, data_element, pixel_crop=None):
+    def _load_as_matrix(
+        self, data_element: DataElement,
+            pixel_crop: AxisAlignedBoundingBox = None) -> np.ndarray:
```

Comment:
Nitpick: that's some odd formatting to me.
```suggestion
    def _load_as_matrix(
        self,
        data_element: DataElement,
        pixel_crop: AxisAlignedBoundingBox = None
    ) -> np.ndarray:
```

Reply:
nvm, i'm being pedantic.